Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -212,6 +213,9 @@ - (void)runWithArguments:(NSArray *)arguments {
         @"non_root_cache_count" : @(nonRootCacheCount),
       };
     }
+    if (configurator.blockUSBMount && [configurator.remountUSBMode count] > 0) {
+      stats[@"daemon"][@"remount_usb_mode"] = configurator.remountUSBMode;
```

Comment:
`stats[@"daemon"]` is not mutable. You'll have to either:

a) Change the closing brace on line 192 (where the "daemon" nested dict is closed) to add `mutableCopy]` and an appropriate opening bracket.

b) Create this in-place, under the new block_usb key add something like:

`@"remount_usb_mode": (configurator.blockUSBMount && configurator.remountUSBMode.count ? configurator.remountUSBMode : @""),`

Reply:
Done. I'll create it inplace.